cabby/geo/walk.py

```diff
@@ -208,11 +209,9 @@ def get_pivot_near_goal(map: map_structure.Map, end_point: GeoDataFrame) -> \
     if prominent_poi is None:
         return None
 
-    return prominent_poi.to_dict('records')[0]
-
+    return prominent_poi
 
-def get_pivots(route: GeoDataFrame, map: map_structure.Map, end_point:
-               GeoDataFrame) -> Optional[Tuple[Dict, Dict]]:
+def get_pivot_along_route(route: GeoDataFrame, map: map_structure.Map) -> Optional[Dict]:
```

Comment:
Specify key and value types for Dict.

Reply:
As I look further (e.g. at get_pivots()), perhaps we need a pivot class that contains the relevant info? That would also help as code-as-documentation.

Reply:
Good for now, will consider ways to managed this better in future.